src/_includes/yt_shims.liquid

```diff
@@ -1,6 +0,0 @@
-{% assign yt-set = 'frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen loading="lazy"' %}
```

Comment:
do we define these slugs anywhere else?

Reply:
Answered my own question in yt.yml :)

Reply:
Not in the same way. These were error-prone as it wasn't clear if you forgot to include this file.

Now they're declared as global data in `src/_data/yt.yml`. You can access any of those properties like `{{yt.embed}}` or `{{yt.std-size}}`.